nx-X11/programs/Xserver/hw/nxagent/nxagent.xpm

```diff
@@ -1,49 +1,373 @@
+"B 	c #D5D5D5",
+"C 	c #FEFEFE",
+"D 	c #A9A9A9",
+"E 	c #262626",
+"F 	c #E4E4E4",
+"G 	c #030303",
+"H 	c #EEEEEE",
+"I 	c #050505",
+"J 	c #C3C3C3",
+"K 	c #2E2E2E",
+"L 	c #979797",
+"M 	c #F0F0F0",
+"N 	c #202020",
+"O 	c #101010",
+"P 	c #C9C9C9",
+"Q 	c #4A4A4A",
+"R 	c #626262",
+"S 	c #9A9A9A",
+"T 	c #E9E9E9",
+"U 	c #414141",
+"V 	c #686868",
+"W 	c #A0A0A0",
+"X 	c #E2E2E2",
+"Y 	c #404040",
+"Z 	c #8C8C8C",
+"` 	c #BFBFBF",
+" .	c #191919",
+"..	c #161616",
+"+.	c #BCBCBC",
+"@.	c #373737",
+"#.	c #A3A3A3",
+"$.	c #303030",
+"%.	c #5B5B5B",
+"&.	c #727272",
+"*.	c #F6F6F6",
+"=.	c #595959",
+"-.	c #878787",
+";.	c #DCDCDC",
+">.	c #0A0A0A",
+",.	c #2D2D2D",
+"'.	c #CDCDCD",
+").	c #B5B5B5",
+"!.	c #C2C2C2",
+"~.	c #131313",
+"{.	c #848484",
+"].	c #424242",
+"^.	c #070707",
+"/.	c #535353",
+"(.	c #707070",
+"_.	c #AFAFAF",
+":.	c #969696",
+"<.	c #EAEAEA",
+"[.	c #2A2A2A",
+"}.	c #B4B4B4",
+"|.	c #171717",
+"1.	c #CECECE",
+"2.	c #4F4F4F",
+"3.	c #D1D1D1",
+"4.	c #1E1E1E",
+"5.	c #565656",
+"6.	c #7F7F7F",
+"7.	c #3B3B3B",
+"8.	c #BABABA",
+"9.	c #696969",
+"0.	c #CBCBCB",
+"a.	c #090909",
+"b.	c #E1E1E1",
+"c.	c #9C9C9C",
+"d.	c #AEAEAE",
+"e.	c #C8C8C8",
+"f.	c #060606",
+"g.	c #818181",
+"h.	c #D2D2D2",
+"i.	c #343434",
+"j.	c #5A5A5A",
+"k.	c #0F0F0F",
+"l.	c #828282",
+"m.	c #EDEDED",
+"n.	c #7C7C7C",
+"o.	c #A7A7A7",
+"p.	c #CFCFCF",
+"q.	c #1F1F1F",
+"r.	c #E8E8E8",
+"s.	c #A6A6A6",
+"t.	c #6E6E6E",
+"u.	c #7A7A7A",
+"v.	c #6D6D6D",
+"w.	c #B9B9B9",
+"x.	c #0C0C0C",
+"y.	c #B6B6B6",
+"z.	c #333333",
+"A.	c #9D9D9D",
+"B.	c #FCFCFC",
+"C.	c #808080",
+"D.	c #6F6F6F",
+"E.	c #3F3F3F",
+"F.	c #868686",
+"G.	c #444444",
+"H.	c #E3E3E3",
+"I.	c #DBDBDB",
+"J.	c #ADADAD",
+"K.	c #141414",
+"L.	c #A1A1A1",
+"M.	c #2B2B2B",
+"N.	c #111111",
+"O.	c #EFEFEF",
+"P.	c #E5E5E5",
+"Q.	c #434343",
+"R.	c #B1B1B1",
+"S.	c #515151",
+"T.	c #BEBEBE",
+"U.	c #1D1D1D",
+"V.	c #939393",
+"W.	c #898989",
+"X.	c #F1F1F1",
+"Y.	c #2C2C2C",
+"Z.	c #D4D4D4",
+"`.	c #AAAAAA",
+" +	c #383838",
+".+	c #7E7E7E",
+"++	c #4D4D4D",
+"@+	c #8D8D8D",
+"#+	c #5C5C5C",
+"$+	c #E0E0E0",
+"%+	c #BBBBBB",
+"&+	c #1A1A1A",
+"*+	c #353535",
+"=+	c #575757",
+"-+	c #363636",
+";+	c #ABABAB",
+">+	c #B3B3B3",
+",+	c #8E8E8E",
+"'+	c #B0B0B0",
+")+	c #BDBDBD",
+"!+	c #858585",
+"~+	c #3D3D3D",
+"{+	c #616161",
+"]+	c #484848",
+"^+	c #3E3E3E",
+"/+	c #B7B7B7",
+"(+	c #585858",
+"_+	c #A4A4A4",
+":+	c #909090",
+"<+	c #474747",
+"[+	c #DADADA",
+"}+	c #494949",
+"|+	c #676767",
+"1+	c #C5C5C5",
+"2+	c #777777",
+"3+	c #C4C4C4",
+"4+	c #242424",
+"5+	c #606060",
+"6+	c #646464",
+"7+	c #D0D0D0",
+"8+	c #8F8F8F",
+"9+	c #838383",
+"0+	c #292929",
+"a+	c #DEDEDE",
+"b+	c #D6D6D6",
+"c+	c #555555",
+"d+	c #DDDDDD",
+"e+	c #212121",
+"f+	c #949494",
+"g+	c #CACACA",
+"h+	c #6B6B6B",
+"i+	c #656565",
+"j+	c #919191",
+"k+	c #8B8B8B",
+"l+	c #545454",
+"m+	c #C0C0C0",
+"n+	c #A5A5A5",
+"o+	c #525252",
+"p+	c #454545",
+"q+	c #9E9E9E",
+"r+	c #C1C1C1",
+"s+	c #DFDFDF",
```

Comment:
It looks to me that there's too much information in that xpm. Compare to the original nx xpm.

Reply:
Also: Why not generate the xpm on the fly during compilation?

Reply:
I agree that there is space for improvement regarding the XPM generation and design. I am happy to see more work done on that. Possibly, I will add more work myself. But I heard that Ionic is not content with the artwork and possibly might take a look. Let's see.